### DISCUSSION TITLE
yajra datatable as a livewire component not working

### DISCUSSION THREAD

**Discussion by kengheng1024** [yajra/laravel-datatables#3102] [Livewire Integration]
I've tested using below syntax at datatable controller with drawCallback.
return Livewire::mount('resend', ['email' => $row->email])->html();

The livewire working, but my livewire component has a listener, when emit from blade, it will call the listener multiple time with the number of records display.
eg: If have 3 records, click either one of the record, it will fire 3 times.

> I've included the yajra datatable as a livewire components, however, the livewire action is not triggerred. 
> 
> I've tested the livewire action out of the yajra datatable, is working.
> 
> Anyone has tested success?
> 
> ![yajra_datatable_livewire](https://github.com/yajra/laravel-datatables/assets/26240069/d35ac323-2137-4ca3-a4f8-737ce9ddf97c)

**Comment by yajra** [comment]
On [drawCallback](https://datatables.net/reference/option/drawCallback), include `livewire.rescan()`

```js
function(settings) {
  if (window.livewire) {
      window.livewire.rescan();
  }
}
```
